### PR TITLE
Update getDefaultDict to return CmdDict extensions when applicable

### LINKED ISSUE #328: CmdDict extensions aren't possible
Using `cmd.getDefaultDict` does not accommodate loading a custom `CmdDict` extension.

```
def getDefaultDict(reload=False):
    return util.getDefaultDict(__name__, 'cmddict', CmdDict, reload)
```

The `CmdDict` extension "works" in that `utils.__init_extensions__` creates the `createCmdDict` function so the usual extension magic cooperates. Unfortunately, pretty much everything in the toolkit uses `getDefaultDict` so it's pretty much useless.

### PR DESCRIPTION
**Pull request by jasonmlkang**
Fix #328

### CODE REVIEW

**Review by nttoole — CHANGES_REQUESTED** [review]
@jasonmlkang Could you please add a unit test or two to exercise this?  You can refer to an earlier PR on how they approached testing extensions:  https://github.com/NASA-AMMOS/AIT-Core/pull/393/files
Thanks

**Comment by jasonmlkang** [comment]
> @jasonmlkang Could you please add a unit test or two to exercise this? You can refer to an earlier PR on how they approached testing extensions: https://github.com/NASA-AMMOS/AIT-Core/pull/393/files Thanks

@nttoole Added unit test in 4cd31ca

File: config/cmd.yaml [hunk 1]
```diff
@@ -84,4 +84,4 @@
       bytes: [0,15]
 
 # below is example of including additional command dictionary file
-- !include core_set_op_mode.yaml
+# - !include core_set_op_mode.yaml
```
Comment (nttoole) [inline config/cmd.yaml:87]: @jasonmlkang I know you filed a separate bug issue for this, but just wanted to understand it better.  Does this include work with the standard CmdDict use case?

Comment (jasonmlkang) [reply config/cmd.yaml:87]: @nttoole It does not work with the default CmdDict specified in default config yaml.

**Comment by jasonmlkang** [comment]
@nttoole Fixed unit tests in [d6b088d](https://github.com/NASA-AMMOS/AIT-Core/pull/404/commits/d6b088d49da6227ff77df953f2dbd94793af2e72)

**Review by nttoole — APPROVED** [review]

**Pull request merged**